docs/tips/03-if-else-in-JSX.md

```diff
@@ -30,7 +30,7 @@ React.createElement("div", {id: if (condition) { 'msg' }}, "Hello World!");
 That's not valid JS. You probably want to make use of a ternary expression:
 
 ```js
-ReactDOM.render(<div id={condition ? 'msg' : ''}>Hello World!</div>, mountNode);
+ReactDOM.render(<div id={condition ? 'msg' : null}>Hello World!</div>, mountNode);
```

Comment:
Ah you're right, I didn't read the full example. It seems to me that `null` or `false` is a better convention than `''`, as the behavior is the same aside from `''` creating extra `<span>` tags.

Perhaps it would be better yet if empty strings were treated like null but I am not sure if that is possible.